deployments/helm/configmapfiles/discovery-engine/conf.yaml

```diff
@@ -35,10 +35,11 @@ observability:
   system-observability: true
   network-observability: false
   write-logs-to-db: false
+  write-to-db: false
+  summary-cron-interval: "0h1m00s"

```

Comment:
We can rename this as `db-write-interval`.

Reply:
We are not doing db operation alone based on this flag! We are publishing SysSummary to gRPC!